docs/docsite/rst/user_guide/intro_getting_started.rst

```diff
@@ -109,6 +109,12 @@ You can read more about privilege escalation in :ref:`become`.
 
 Congratulations! You have contacted your nodes using Ansible. You used a basic inventory file and an ad-hoc command to direct Ansible to connect to specific remote nodes, copy a module file there and execute it, and return output. You have a fully working infrastructure.
 
+Resources
+=================================
+
+* https://github.com/ansible/product-demos
```

Comment:
Add with URL Format:

```md
* [Product Demos](https://github.com/ansible/workshops)
* [Katakoda](https://katacoda.com/rhel-labs)
```
* [Product Demos](https://github.com/ansible/workshops)
* [Katakoda](https://katacoda.com/rhel-labs)

Reply:
Sorry @kshitijcode I added formatting for `md` and not for `rst`. Could you please review? Thank you